lib/color_consts.dart

```diff
@@ -0,0 +1,9 @@
+
+import 'dart:ui';
+
+const Color kBackground1Color= Color(0xff005a44);
+const Color kBackground2Color= Color(0xfffeefe5);
+const Color kFirstColor= Color(0xffffcf00);
+const Color kSecondColor= Color(0xffee6123);
+const Color kThirdColor= Color(0xfffa003f);
+const Color kButtonColor= Color(0xff00140f);
```

Comment:
More color related names of colors would be nice like: kPrimaryOrange or kBackgroundBlack. It would make deciding between them easier also stick to primary secondary rather than first second naming

Reply:
Ready